history/cultures/undead_group.txt

```diff
@@ -42,6 +42,8 @@
 	discover_innovation = innovation_hereditary_rule 
 	discover_innovation = innovation_royal_prerogative 
 	#
+	discover_innovation = innovation_undead_ground
```

Comment:
I was just following the pattern I used of giving special maa's around 270, but it would make sense for undead to get these right away...

Reply:
> I was just following the pattern I used of giving special maa's around 270, but it would make sense for undead to get these right away...

That's what I though.